packages/docprovider/src/yprovider.ts

```diff
@@ -216,43 +127,19 @@ export class WebSocketProviderWithLocks
     send();
   }
 
-  /**
-   * Handle a change to the connection status.
-   *
-   * @param status The connection status.
-   */
-  private async _onConnectionStatus(status: {
-    status: 'connected' | 'disconnected';
-  }): Promise<void> {
-    if (this._isInitialized && status.status === 'connected') {
-      const lock = await this.acquireLock();
-      const contentIsInitialized = await this.requestInitialContent();
-      if (!contentIsInitialized) {
-        this.putInitializedState();
-      }
-      this.releaseLock(lock);
-    }
-  }
-
   private _path: string;
```

Comment:
I would like to understand why the format and contentType are needed in this API.

Reply:
I think it's because jupyter-server's contents manager [needs them](https://github.com/jupyter-server/jupyter_server_ydoc/blob/201bd2ebdb8e618d07f0f1125b746c51007ffe19/jupyter_server_ydoc/ydoc.py#L148).